install.py

```diff
@@ -332,7 +332,7 @@ def install(inst, version):
     elif version is None:  # install the stable version
         subprocess.check_call([pycmd, '-m', 'pip', 'install',
                                '--upgrade', 'openquake.engine'])
-    elif re.match(version, r'\d+(\.\d+)+'):  # install an official version
+    elif re.match(r'(\d+)\.(\d+)', version): # install an official version
         subprocess.check_call([pycmd, '-m', 'pip', 'install',
```

Comment:
This is wrong, for instance you cannot install 3.11.4. What was the problem with the regex?

Reply:
If I use the follow: python3 install.py user --version=3.11 I have this error:
Traceback (most recent call last): 
 File "install.py", line 463, in <module> 
   install(inst, args.version) 
 File "install.py", line 339, in install 
   commit = latest_commit(version) 
 File "install.py", line 252, in latest_commit 
   with urlopen(url) as f: 
 File "/usr/lib/python3.6/urllib/request.py", line 223, in urlopen 
   return opener.open(url, data, timeout) 
 File "/usr/lib/python3.6/urllib/request.py", line 532, in open 
   response = meth(req, response) 
 File "/usr/lib/python3.6/urllib/request.py", line 642, in http_response 
   'http', request, response, code, msg, hdrs) 
 File "/usr/lib/python3.6/urllib/request.py", line 570, in error 
   return self._call_chain(*args) 
 File "/usr/lib/python3.6/urllib/request.py", line 504, in _call_chain 
   result = func(*args) 
 File "/usr/lib/python3.6/urllib/request.py", line 650, in http_error_default 
   raise HTTPError(req.full_url, code, msg, hdrs, fp) 
urllib.error.HTTPError: HTTP Error 422: Unprocessable Entity